README.md

```diff
@@ -1,5 +1,6 @@
 To start hacking on Hydra, run:
 
+    $ bootstrap
```

Comment:
Good point! Should we call bootstrap from release.nix or remove the bootstrap script? I don't like having the autoreconf call duplicated :)